Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -177,10 +210,28 @@ public override Ust VisitOmittedArraySizeExpression(OmittedArraySizeExpressionSy
 
         public override Ust VisitAssignmentExpression(AssignmentExpressionSyntax node)
         {
+            var textSpan = node.GetTextSpan();
+            if (node.Left is TupleExpressionSyntax tupleLeft
+                && node.Right is TupleExpressionSyntax tupleRight)
+            {
+                var assignments = new List<AssignmentExpression>();
```

Comment:
For the future: if the number of elements is defined, initialize the list with it: `var assignments = new List<AssignmentExpression>(tupleLeft.Arguments.Count);`